packages/amplify-cli-core/tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "outDir": "./lib",
     "rootDir": "./src",
+    "useUnknownInCatchVariables":false
```

Comment:
won't this get pulled in from the base tsconfig?

Reply:
somehow it isnt , I tried before